crates/typst-html/src/rules.rs

```diff
@@ -445,7 +449,75 @@ const OUTLINE_ENTRY_RULE: ShowFn<OutlineEntry> = |elem, engine, styles| {
 
 const REF_RULE: ShowFn<RefElem> = |elem, engine, styles| elem.realize(engine, styles);
 
-const CITE_GROUP_RULE: ShowFn<CiteGroup> = |elem, engine, _| elem.realize(engine);
+const CITE_GROUP_RULE: ShowFn<CiteGroup> = |elem, engine, _| {
+    Ok(elem
+        .realize(engine)?
+        .styled(HtmlElem::role.set(Some("doc-biblioref".into()))))
+};
+
+// For the bibliography, we have a few elements that should be styled (e.g.
+// indent), but inline styles are not apprioriate because they couldn't be
+// properly overridden. For those, we currently emit classes so that a user can
+// style them with CSS, but do not emit any styles ourselves.
+const BIBLIOGRAPHY_RULE: ShowFn<BibliographyElem> = |elem, engine, styles| {
+    let span = elem.span();
+    let works = Works::generate(engine).at(span)?;
+    let references = works.references(elem, styles)?;
+
+    let items = references.iter().map(|(prefix, reference, loc)| {
+        let mut realized = reference.clone();
+
+        if let Some(mut prefix) = prefix.clone() {
+            // If we have a link back to the first citation referencing this
+            // entry, attach the appropriate role.
+            if prefix.is::<DirectLinkElem>() {
+                prefix = prefix.set(HtmlElem::role, Some("doc-backlink".into()));
+            }
+
+            let wrapped = HtmlElem::new(tag::span)
+                .with_attr(attr::class, "prefix")
```

Comment:
Is the prefix class here just intended to allow the user to style the outputted HTML? It seems like a pretty generic class name to me that could easily conflict/collide with existing stylesheets if the generated HTML is embedded in an existing page, say. 

Would it make sense to provide a Typst prefix (excuse the pun) for class names so that this becomes something like `typst__prefix`? Maybe this is best exposed as an option when passing the HTML output flag, as I realize it might be overkill for standalone documents.